src/pages/list/List.js

```diff
@@ -0,0 +1,326 @@
+import React, { useState, useEffect } from "react";
+import DatePicker from "react-datepicker";
+import styled from "styled-components";
+import { ko } from "date-fns/esm/locale";
+
+const List = () => {
+  const [startDate, setStartDate] = useState(new Date());
+  const [themes, setThemes] = useState();
+  const [roomList, setRoomList] = useState();
```

Comment:
어떤 형태의 데이터를 가지는지 알 수 있게 초기값도 넣어주시는 게 좋을 것 같습니다.
```suggestion
  const [themes, setThemes] = useState([]);
  const [roomList, setRoomList] = useState([]);
```